ocaml/testsuite/tests/typing-unique/overwriting_underscore.ml

```diff
@@ -0,0 +1,24 @@
+(* TEST
+   expect;
+*)
+
+(* For backwards-compatibility, we want to keep the same error message when the extension
```

Comment:
No need to be backward compatible in this way. That said, the error message is fine as it is.